docker/docker-compose.ci.yml

```diff
@@ -53,17 +56,20 @@ services:
     depends_on:
       - ganache-integration
       - unlock
+      - locksmith
```

Comment:
I think we should not have it both here and in `unlock` above. I'm in favor of just having it here but that's not a huge deal either way.